src/ocrd/resource_manager.py

```diff
@@ -99,10 +99,17 @@ def list_available(self, executable=None, dynamic=True, name=None, database=None
         if not executable:
             return database.items()
         if dynamic:
+            skip_executables = ["ocrd-cis-data", "ocrd-import", "ocrd-make"]
             for exec_dir in environ['PATH'].split(':'):
                 for exec_path in Path(exec_dir).glob(f'{executable}'):
+                    if not exec_path.name.startswith('ocrd-'):
+                        self.log.debug(f"Not in 'ocrd-*' format, skipping '{exec_path}'")
+                        continue
```

Comment:
If the user explicitly requested an executable not starting with `ocrd-`, then IMO we should not skip that (but let it fail).

Reply:
Or perhaps issue a warning?
```suggestion
                    if not exec_path.name.startswith('ocrd-'):
                        self.log.warning(f"OCR-D processor executable '{exec_path}' has no 'ocrd-' prefix")
```